HW6/Andrii_Haponov/hw_6_3.py

```diff
@@ -0,0 +1,27 @@
+# Task3. Write a function that calculates the number of characters included in a given string
+
+# input: “hello”
+# output: {“h”:1, “e”:1, “l”:2, “o”:1}
+
+# Задача3. Напишите функцию, которая вычисляет количество символов, содержащихся в заданной строке.
+
+# ввод: “hello”
+# вывод: {"h":1, "e":1, "l":2, "o":1}
+
```

Comment:
here is a more optimal code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result